### PR TITLE
ZFS SNMP agent :3 <3

### PR DESCRIPTION
**Pull request by VVelox**
Wooho!

Sorry about only having a FreeBSD one at this time. But it is ready to go.

### CODE REVIEW

File: snmp/zfs-freebsd [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+Pretty print the JSON.
+
+=head1 SNMPD SETUP EXAMPLES
+
+    extend zfs /etc/snmp/zfs-freebsd
+
+=cut
+
+#Copyright (c) 2017, Zane C. Bowers-Hadley
```
Comment (laf) [inline snmp/zfs-freebsd]: We need a GPLv3 compatible license, I can't tell what license this is from the below.

Comment (VVelox) [reply snmp/zfs-freebsd]: that is the standard 2 clause BSDL

Comment (VVelox) [reply snmp/zfs-freebsd]: https://www.gnu.org/licenses/license-list.en.html#FreeBSD

That with "The views and conclusions contained in the software and documentation are those of the authors and should not be interpreted as representing official policies, either  expressed or implied, of the FreeBSD Project." removed from the end.

**Comment by enilfodne** [comment]
Sorry in advance to discuss this in the PR, but i don't want to open an issue if its intended behavior.

I was wondering if the output length might be a problem?

I have 2 FreeBSD machines and the output from `zfs-freebsd` is ~1500 and ~1800 characters, respectively. However `NET-SNMP` times-out when the `zfs` extend is enabled on the second machine (~1800 chars), while it works fine on the first one (~1500).

Connection is UDP/1500MTU

**Pull request merged**